ja:
　errors:
      messages:
       not_saved: "セーブに失敗しました"
  